src/main/resources/configuration/serverconf

```diff
@@ -0,0 +1 @@
+server1 = 10.0.0.202:1993
```

Comment:
Is it possible to add this to application.properties file instead of creating a new config file?